tests/framework/OutStreams/tests

```diff
@@ -87,4 +87,17 @@
     input = 'sample_plot.xml'
     output = 'SamplePlot/plotter.png'
   [../]
+  
+  [./PopulationPlot]
+    type = 'RavenFramework'
+    input = 'ga_population_plot.xml'
+    output = 'GApopulation/plotter.png'
+  [../]
+  
+  [./ParallelCoordinatePlot]
+    type = 'RavenFramework'
+    input = 'parallel_coordinate_plot.xml'
+    output = 'GApopulation/plotter1.png'
```

Comment:
please use 'image =' key word to check the diff

Reply:
fixed